src/jit/gentree.h

```diff
@@ -2671,7 +2671,7 @@ struct GenTreeCall final : public GenTree
 
 #ifdef FEATURE_READYTORUN_COMPILER
     // Call target lookup info for method call from a Ready To Run module
-    CORINFO_CONST_LOOKUP gtEntryPoint;
+    CORINFO_LOOKUP gtEntryPoint;
```

Comment:
It is much bigger than the CORINFO_CONST_LOOKUP but this is only added for READYTORUN_COMPILER feature defined. As far as I know this is mostly for crossgen a nd never used at runtime jitting. Is that right?